plugins/policy-recommendation/policy_recommendation_job.py

```diff
@@ -0,0 +1,839 @@
+    id = ""
+    rm_labels = True
+    to_services = True
+    help_message = """
+    Start the policy recommendation spark job.
+
+    Options:
+    -h, --help: Show help message.
+    -t, --type=initial: {initial|subsequent} Indicates this recommendation is an initial recommendion or a subsequent recommendation job.
+    -d, --db_jdbc_url=None: The JDBC URL used by Spark jobs connect to the ClickHouse database for reading flow records and writing results.
+        jdbc:clickhouse://clickhouse-clickhouse.flow-visibility.svc:8123 is the ClickHouse JDBC URL used by default.
+    -l, --limit=0: The limit on the number of flow records read from the database. 0 means no limit.
+    -o, --option=1: Option of network isolation preference in policy recommendation.
+        Currently we have 3 options:
+        1: Recommending allow ANP/ACNP policies, with default deny rules only on applied to Pod labels which have allow rules recommended.
+        2: Recommending allow ANP/ACNP policies, with default deny rules for whole cluster.
+        3: Recommending allow K8s network policies, with no deny rules at all.
+    -s, --start_time=None: The start time of the flow records considered for the policy recommendation. 
+        Format is YYYY-MM-DD hh:mm:ss in UTC timezone. Default value is None, which means no limit of the start time of flow records.
+    -e, --end_time=None: The end time of the flow records considered for the policy recommendation.
+        Format is YYYY-MM-DD hh:mm:ss in UTC timezone. Default value is None, which means no limit of the end time of flow records.
+    -n, --ns_allow_list=[]: List of default traffic allow namespaces.
+        Default value is a list of Antrea CNI related namespaces: ['kube-system', 'flow-aggregator', 'flow-visibility'].
+    -i, --id=None: Recommendation job ID in UUID format. If not specified, it will be generated automatically.
+    --rm_labels=None: Remove automatically generated Pod labels including 'pod-template-hash', 'controller-revision-hash', 'pod-template-generation'.
+        This feature is enabled by default, provide false to disable this feature.
+    --to_services=None: Use the toServices feature in ANP and recommendation toServices rules for Pod-to-Service flows, only works when option is 1 or 2.
+        This feature is enabled by default, provide false to disable this feature.
+    
+    Usage Example:
+    python3 policy_recommendation_job.py -t initial -l 1000 -o 1 -s '2021-01-01 00:00:00' -e '2021-12-31 00:00:00' -n '["kube-system","flow-aggregator","flow-visibility"]'
+    """
+
+    try:
+        opts, _ = getopt.getopt(argv, "ht:d:l:o:s:e:n:i:", ["help", "type=", "db_jdbc_url=", "limit=", "option=", "start_time=", "end_time=", "ns_allow_list=", "id=", "rm_labels=", "to_services="])
+    except getopt.GetoptError as e:
+        print("getopt.getopt ERROR: {}".format(e))
+        print(help_message)
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt in ("-h", "--help"):
```

Comment:
I'd check for -h before entering the loop, a user my specify also other options, we don't want to parse them if -h is specified.

Reply:
Sounds good to me, thanks!